src/maintainer/maintainer_faq.rst

```diff
@@ -71,20 +71,11 @@ FAQ
 
 .. _mfaq_mamba_local:
 
-:ref:`(Q) <mfaq_mamba_local>` **How can I make debugging solver failures on the CI faster?**
+:ref:`(Q) <mfaq_mamba_local>` **How can I make local debugging faster?**
 
-  An alternative to the conda solver is the mamba solver. The mamba solver has a faster solve speed and prints better error messages that make debugging simpler.
+  If you prefer to debug your recipes locally and not use the provided `scripts <https://conda-forge.org/docs/maintainer/updating_pkgs.html#testing-changes-locally>`__ but instead your own setup, you may also use the mamba solver through ``mambabuild``. It not only has a faster solve speed but also prints better error messages that make debugging simpler.
```

Comment:
just a quick note: using sphinx' internal links with `:ref:` instead of external links prevents accidentally breaking links when renaming sections 😄

Reply:
Ahh TIL. Happy to create a follow-up or do you want to include that in https://github.com/conda-forge/conda-forge.github.io/pull/1565

Reply:
It's not broken, so I'd say fix whereever/whenever you feel it's appropriate 👍

Reply:
It's not broken, so I'd say change where ever/whenever you feel it's appropriate 👍